enhancements/api-review/custom-tags-aws.md

```diff
@@ -73,71 +95,397 @@ Note existing unchanged behavior: The installer will apply these tags to all AWS
+
+   New tag added in kube resource object like machine CR = `key_infra1 = value1`
+
+   Action = The tag for AWS resource is updated to `key_infra1 = value1`.
+
+   Final tag set to AWS resource = `key_infra1 = value1`
+
+   Event action = An event is generated to notify user about the request status (success) to update tags for the AWS resource. A warning also must be generated about list of user-defined tags on which action did a override.
+
+3) `.spec.platformSpec.aws` take precedence over `.status.platformStatus.aws`. User-defined tags must be merged from `.status.platformStatus.aws` and set to AWS resource.
+
+4) Before deleting user-defined tag from AWS resource, the entry must be removed from `.status.platformStatus.aws`, if any, as operator will reconcile user-defined tag to the AWS resource.
+   Setting user-defined tag to empty string in `.spec.platformSpec.aws` will not override the user-defined tag in `.status.platformStatus.aws`.
+
+#### Caveats
+1) User updates a resource's tag using an external tool when there is an entry in `.spec.platformSpec.aws.resourceTags`
+   The resource's tag will be reconciled by its owning operator to the value from `.spec.platformSpec.aws.resourceTags`.
+
+   For example,
+
+   Edited existing tag using external tool for AWS resource = `key_infra1 = value_comp1`
+
+   Previous tag request = `.spec.platformSpec.aws.resourceTags` has `key_infra1 = value_infra1`
+
+   Action = Update existing tag with value from `.spec.platformSpec.aws.resourceTags`.
+
+   Final tag set to AWS resource = `key_infra1 = value_infra1`
+
+   Event action = An event is generated to notify user about the request status (success/failure) to update tags for the AWS resource.
+
+2) User deletes the user-defined tag from `.spec.platformSpec.aws.resourceTags`
+   The user-defined tag which is removed from spec will not be reconciled or managed by operators.
+
+   User can update user-defined tag key:value using external tool. The user-defined tag will not be overwritten.
+
+   For example,
+
+   Existing tag for AWS resource = `key_infra1 = value1`
+
+   New tag request = `.spec.platformSpec.aws.resourceTags` has no user-defined tag with key `key_infra1`
+
+   Action = No change in tags for the AWS resource.
+
+   Final tag set to AWS resource = `key_infra1 = value1`
+
+   Event action = No event
+
+3) User sets user-defined tag created using external tools to empty string for deleting for AWS resource.
+
+   For example,
+
+   User add tag using external tools (or directly) on AWS resource = `key_infra1 = value1`
+
+   New tag request = None
+
+   Action = No update done for tags for the AWS resource by the owning operator.
+
+   Final tag set to AWS resource = `key_infra1 = value1`
+
+   Event action = No event
+
+   New tag request = `.spec.platformSpec.aws.resourceTags` has `key_infra1 =`
+
+   Action = No change in tags for the AWS resource.
+
+   Final tag set to AWS resource = `deleted
```

Comment:
You've missed a closing backtick on this line which is messing up your formatting below

Reply:
Thanks for locating it.